src/components/MultiHopTrade/MultiHopTrade.tsx

```diff
@@ -31,11 +37,17 @@ const TradeRouteEntries = [
 export type TradeCardProps = {
   defaultBuyAssetId?: AssetId
   isCompact?: boolean
+  isRewritingUrl?: boolean
 }
 
 type MatchParams = {
   chainId?: string
   assetSubId?: string
+  fromAssetSubId?: string
+  fromChainId?: string
+  sellAssetSubId?: string
+  sellChainId?: string
+  sellAmountCryptoBaseUnit?: string
```

Comment:
Obviously not for this PR, but noting we still use the subId/chainId parts. We *should* be able to safely remove this and make things a lot simpler in a follow-up if you feel like giving it a go!

For historical context, before the removal of the splash screen, there was a redirectUrl, which was the URL the app would redirect to after app refresh on splash screen and wallet connection. This redirectUrl would serialize weirdly i.e with weird UTF-8 encoding, and we couldn't use regular parts there. 

With that gone, I *believe* we should be able to use CAIPs as-is, without parts.

Reply:
Not 100% sure as caip contains `/` which would be parsed by react-router as param separator